### DISCUSSION TITLE
Is this functional and is there a way to deploy a lazer compatible server with this?

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [Pure-Peace/peace#4] [Q&A] [answered]
as the title asks, is this functional and how much is complete? Can this be used to host a lazer compatible private server and how would you go about connecting with the lazer client? Is this it's own ecosystem entirely or can this be integrated into an existing ripple style server?

**Comment by Pure-Peace** [comment]
This project currently only implements bancho. And, most of the features are done except multiplayer.

I've been thinking about and preparing to refactor this project to make it a pluggable server project.

**Comment by auroraongithub** [comment]
when is the tutorial and documation coming?

**Reply by Pure-Peace** [reply]
> when is the tutorial and documation coming?

Project have been completely redesigned.  

The unfinished building will be demolished and rebuilt...

**Comment by Pure-Peace** [comment, marked as answer]
Project have been completely redesigned.

The unfinished building will be demolished and rebuilt...
